### DISCUSSION TITLE
[BUG] Get-PnPTeamsUser: Exception of type 'PnP.PowerShell.Commands.Model.Graph.GraphException' was thrown

### DISCUSSION THREAD

**Discussion by YuriySamorodov** [pnp/powershell#2446] [General] [answered]
### Reporting an Issue or Missing Feature
I am reporting a bug

### Expected behavior 
`$members = Get-PnPTeamsUser -Team Team.Test` should not return any errors

### Actual behavior 
```
 Get-PnPTeamsUser -Team Team.GlobalInformationSecurity
Get-PnPTeamsUser: Exception of type 'PnP.PowerShell.Commands.Model.Graph.GraphException' was thrown
```
![image](https://user-images.githubusercontent.com/5260172/194905367-f3a6f8d0-86a7-4f67-9c74-d9a7a211557d.png)

### Steps to reproduce behavior
```
Connect-PnPSite $adminUrl
Get-PnPTeamsTeam -Team Team
```

### What is the version of the Cmdlet module you are running?
```
Name    : PnP.PowerShell
Version : 1.11.99
```

### Which operating system/environment are you running PnP PowerShell on?
- [x] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

**Comment by gautamdsheth** [comment, marked as answer]
Hi @YuriySamorodov , same here as well.

Looks like the app you are using is missing permissions. Please ensure that it has necessary permissions to request data.

![image](https://user-images.githubusercontent.com/8832018/194940319-18acd201-3098-406f-bcf8-6df4089c1445.png)

**Comment by YuriySamorodov** [comment]
Thanks @gautamdsheth . According to the docs https://pnp.github.io/powershell/cmdlets/Get-PnPTeamsUser.html `Get-PnPTeamsUser` requires the following permissions:
* Microsoft Graph API : One of Group.Read.All, Group.ReadWrite.All
* Microsoft Graph API : Directory.Read.All
